### DISCUSSION TITLE
Does this software support Android devices?

### DISCUSSION THREAD

**Discussion by Hhsy083212** [vkbo/novelWriter#2112] [Q&A] [answered]
Android is the world's most widely used operating system, with many tablets and foldable smartphones running on it. I kindly request the developers to consider porting this software to the Android platform. This would benefit a large number of users. Thank you very much!

**Comment by vkbo** [comment, marked as answer]
It does not, and there are no plans to do so.

Access on other platforms than Linux and Windows requires contributions. It is a lot of work to adopt the app to all these platforms, and I can't do it all. Someone fixed it for MacOS once, but then left it to me to maintain, so I won't do that again. It adds too much to my workload.

**Reply by vkbo** [reply]
That said, both Python and the framework novelWriter runs on is available on Android, so there's nothing stopping people from just running the app there, or package it for Android.

**Reply by rumplestilzken** [reply]
> That said, both Python and the framework novelWriter runs on is available on Android, so there's nothing stopping people from just running the app there, or package it for Android.

If I were to put in the work to get it running on android would you support and integrate that enhancement?

**Reply by rumplestilzken** [reply]
> That said, both Python and the framework novelWriter runs on is available on Android, so there's nothing stopping people from just running the app there, or package it for Android.

I'm not talking about dropping it on you. I'm asking if you will allow a pull request and keep me in the loop so I can maintain the android portions.

**Reply by vkbo** [reply]
Yes, potentially, depending on what needs to be changed or added. There is a feature request for it: #2144. The topic can be discussed there, or you can open a new discussion thread if you think figuring out a lot of details is needed (I have no idea what this would require). Since this discussion has been answered, I think it's better to move it to one or both of the above.

**Reply by rumplestilzken** [reply]
> Yes, potentially, depending on what needs to be changed or added. There is a feature request for it: #2144. The topic can be discussed there, or you can open a new discussion thread if you think figuring out a lot of details is needed (I have no idea what this would require). Since this discussion has been answered, I think it's better to move it to one or both of the above.

Sure. No problem. Thanks for the reply.
